src/chrome/content/rules/Scssoft.com.xml

```diff
@@ -0,0 +1,17 @@
+<ruleset name="scssoft.com">
```

Comment:
Cloud you also add [comments for non-working subdomains](https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#non-working-hosts)?
```
huntingunlimited.scssoft.com
karolina.scssoft.com
www2.scssoft.com
```